components/ChapterBlock.vue

```diff
@@ -90,6 +90,15 @@ export default {
   computed: {
     sections () {
       return chapter => {
```

Comment:
how about destructuring chapter? then you can use just `frontmatter` instead of `chapter.frontmatter`
```suggestion
      return ({frontmatter}) => {
```

Reply:
I can't do it in that place I thin
Just after if statement, I need chapter object keys like level and headers in case when `chapter_home_intro` is not provided.

Reply:
uh, sorry haven't noticed that, so maybe do 
```suggestion
      return ({ frontmatter, headers }) => {
``` 
and for line 102
```suggestion
      const chapterSections = headers && headers.filter(({ level }) => level === 2)
```

Reply:
Sure, that would work 👍  I've just pushed that change